src/components/shared/experimental-blockquote/experimental-blockquote.view.js

```diff
@@ -7,8 +7,9 @@ const ExperimentalBlockquote = () => (
     need to add features or introduce breaking changes. This could happen at any
     time until we release this module as stable.
     <p>
-      Feel free to provide user feedback, and open an issue or pull request if
-      you have any suggestions.
+      Feel free to [provide user feedback](https://community.k6.io/), and [open
+      an issue](https://github.com/grafana/k6/issues) or pull request if you
+      have any suggestions.
```

Comment:
I don't know if it won't be better if this is linking to the xk6-websockets repo for now :thinking:

Reply:
I don't have a strong opinion :shrug: as long as we have an established channel of getting feedback :relaxed: 

But here are my thoughts. In theory, we can link to the module's GitHub, but on the other side, we offer it as part of the k6, so issues/feature requests to the main k6 repo look fine. Also, this block is a general block for all experimental extensions.